test/azure/version-tolerant/Expected/AcceptanceTests/CustomUrlPagingVersionTolerant/custombaseurlpagingversiontolerant/aio/operations/_operations.py

```diff
@@ -27,8 +28,13 @@
     build_paging_get_pages_partial_url_request,
 )
 
+try:
+    JSONObject = MutableMapping[str, Any]  # pylint: disable=E1136
+except TypeError:
+    from typing import MutableMapping  # pylint: disable=W0404, C0412
+
+    JSONObject = MutableMapping[str, Any]  # type: ignore # pylint: disable=E1136
```

Comment:
Hey @iscai-msft , help take a look at the generated code, I find have to add these disable rules to get the pylint and mypy green.

Have not updated the customization part yet, I will investigate it later.

Reply:
we shouldn't need all of those pylint and type disables. Can you try this instead, using the following code I only expect one type ignore

```python
if sys.version_info >= (3,9):
  from collections.abc import MutableMapping
else:
  from typing import MutableMapping # type: ignore
JSONObject = MutableMapping[str, Any]
```

Also in the future can you use the actual names for the pylint disables, instead of the codes? I.e. `pylint: disable=protected-access`.

Reply:
Yes, that's better!  
I updated to generate as below, adding pylint disable because the error reported in python<3.9.
~~~
if sys.version_info >= (3, 9):
    from collections.abc import MutableMapping
else:
    from typing import MutableMapping  # type: ignore
JSONObject = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
~~~